addons/mod_tool/templates/default/mod_main.gd

```diff
@@ -11,43 +11,40 @@ extends Node
 
 # ? Brief overview of what your mod does...
 
-const MOD_DIR := "AuthorName-ModName" # Name of the directory that this file is in
-const MODNAME_LOG_NAME := "AuthorName-ModName" # Full ID of the mod (AuthorName-ModName)
+const AUTHORNAME_MODNAME_DIR := "AuthorName-ModName" # Name of the directory that this file is in
+const AUTHORNAME_MODNAME_LOG_NAME := "AuthorName-ModName" # Full ID of the mod (AuthorName-ModName)
```

Comment:
should we add `:Main` at the end to discern between the files? We're doing ti for the mod loader, why not encourage it for mods as well

Reply:
Done 👍 - 55a2cf1